Xamarin.Forms.Platform.iOS/Renderers/EditorRenderer.cs

```diff
@@ -149,8 +172,35 @@ void UpdateKeyboard()
 		void UpdateText()
 		{
 			// ReSharper disable once RedundantCheckBeforeAssignment
+
+			if (string.IsNullOrWhiteSpace(Element.Text) && !Control.IsFirstResponder)
+			{	
+				UpdatePlaceholder();
+				return;
+			}
+
 			if (Control.Text != Element.Text)
+			{
 				Control.Text = Element.Text;
+				UpdateTextColor();
+			}
+		}
+
+		void UpdatePlaceholder()
+		{
+			UpdatePlaceholderText();
+			UpdatePlaceholderColor();
+		}
+
+		void UpdatePlaceholderText()
+		{
+			Control.Text = Element.Placeholder;
+		}
+
+		void UpdatePlaceholderColor()
+		{
+			if (Element.PlaceholderColor != Color.Default)
```

Comment:
Wont this mean that the placeholder color will be the text color assuming text color is set but not placeholder color? That seems confusing.

Reply:
I think ideally we would use the same color text the Entry uses for placeholder.

Reply:
Where? Im not seeing where you are setting the default placeholder color?

Reply:
@jassmith  I'm calling UpdateTextColor() in OnElementChanged  and if you check in UpdateTextColor function there is check that if the text is null or empty then set the placeholder by calling updateplaceholder function. and placeholder function is setting the color and text for me.